scripts/kani-perf.sh

```diff
@@ -27,7 +27,8 @@ done
 suite="perf"
 mode="cargo-kani-test"
 echo "Check compiletest suite=$suite mode=$mode"
-cargo run -p compiletest -- --suite $suite --mode $mode --no-fail-fast
+cargo run -p compiletest -- --suite $suite --mode $mode --no-fail-fast \
+  --kani-flag="--enable-unstable" --kani-flag="--cbmc-args" --kani-flag="--verbosity" --kani-flag="9"
```

Comment:
Is `--verbosity 9` now necessary to report performance statistics?

Reply:
Yes, CBMC no longer prints timing stats by default.

Reply:
I'll now make `--verbosity 9` the default with Kani.